src/views/Predictions/components/History/helpers.ts

```diff
@@ -0,0 +1,40 @@
+import { Bet, BetPosition } from 'state/types'
```

Comment:
Had to move these helpers into this folder because they are only used by the history pane now and deal with different kind of numbers.